src/databricks/labs/dqx/utils.py

```diff
@@ -326,3 +327,109 @@ def safe_json_load(value: str):
         return json.loads(value)  # load as json if possible
     except json.JSONDecodeError:
         return value
+
+
+def is_file_path(name: str) -> bool:
```

Comment:
please check read_input_data method, it has logic for verifying table or file path. We don't need to provide such a detailed validation